docs/tutorials/python/profiler.md

```diff
@@ -215,29 +206,44 @@ Let's zoom in to check the time taken by operators
 
 The above picture visualizes the sequence in which the operators were executed and the time taken by each operator.
 
-#### 3. View in NVProf
+## Advanced: Using NVIDIA Profiling Tools
 
-You can view all MXNet profiler information alongside CUDA kernel information by using the MXNet profiler along with NVProf.  Use the MXNet profiler as in the samples above, but invoke your python script with the following wrapper process available on most systems that support CUDA:
+MXNet's Profiler is definitely the recommended starting point for profiling MXNet code, but NVIDIA also provides a couple of tools for low-level profiling of CUDA code: NVProf, Visual Profiler and Nsight Compute. You can use these tools to profile all kinds of executables, so they can be used for profiling Python scripts running MXNet. And you can use these in conjunction with the MXNet Profiler to see high-level information from MXNet alongside the low-level CUDA kernel information.
```

Comment:
remove 'definitely' - it's more professional :)

links to NVProf, Visual Profiler, Nsight Compute

Reply:
done.